assets/armsnippets.jsonc

```diff
@@ -140,6 +140,122 @@
+            "          ],",
+            "          \"backendHttpSettingsCollection\": [",
+            "             {",
+            "                \"name\": \"${13:appGatewayBackendHttpSettings}\",",
+            "                \"properties\": {",
+            "                    \"Port\": ${14:80},",
+            "                    \"Protocol\": \"${15|Http,Https|}\",",
+            "                    \"CookieBasedAffinity\": \"Disabled\"",
+            "                }",
+            "            }",
+            "          ],",
+            "          \"httpListeners\": [",
+            "            {",
+            "                \"name\": \"${16:appGatewayHttpListener}\",",
+            "                \"properties\": {",
+            "                  \"FrontendIPConfiguration\": {",
+            "                    \"Id\": \"[resourceId('Microsoft.Network/applicationGateways/frontendIPConfigurations', '${1:applicationGatewayName}', '${8:appGatewayFrontendIP}')]\"",
+            "                  },",
+            "                  \"FrontendPort\": {",
+            "                    \"Id\": \"[resourceId('Microsoft.Network/applicationGateways/frontendPorts',  '${1:applicationGatewayName}', '${9:appGatewayFrontendPort}')]\"",
+            "                  },",
+            "                    \"Protocol\": \"${17|Http,Https|}\",",
+            "                    \"SslCertificate\": {",
+            "                           \"id\": null",
+            "                       }",
+            "                }",
+            "            }",
+            "          ],",
+            "          \"requestRoutingRules\": [",
+            "            {",
+            "                \"Name\": \"${18:ruleName}\",",
+            "                \"properties\": {",
+            "                    \"RuleType\": \"${19|Basic,PathBasedRouting|}\",",
+            "                    \"httpListener\": {",
+            "                    \"id\": \"[resourceId('Microsoft.Network/applicationGateways/httpListeners', '${1:applicationGatewayName}', '${16:appGatewayHttpListener}')]\"",
+            "                  },",
+            "                  \"backendAddressPool\": {",
+            "                    \"id\": \"[resourceId('Microsoft.Network/applicationGateways/backendAddressPools', '${1:applicationGatewayName}', '${10:appGatewayBackendPool}')]\"",
+            "                  },",
+            "                  \"backendHttpSettings\": {",
+            "                    \"id\": \"[resourceId('Microsoft.Network/applicationGateways/backendHttpSettingsCollection', '${1:applicationGatewayName}', '${13:appGatewayBackendHttpSettings}')]\"",
+            "                  }",
+            "                }",
+            "            }",
+            "          ],",
+            "          \"webApplicationFirewallConfiguration\": {",
+            "            \"enabled\": ${19|true,false|},",

```

Comment:
The tabstop identifier her is 19 which is also used above on line 235. As a result, the inserted value is `Basic`. Can you update this tabstop identifier to 20 and then any tabstops found after this one to allign?